README.md

```diff
@@ -30,5 +32,11 @@ cd <sdl_atf>/
 
 __Note, that path to SDL binary dir may be different__
 
+You can get additional help of usage ATF: 
+```
+./start.sh --help
+```
 
-Some test cases are failed due to known SDL issues. List of failed test cases avaliable in [KnownIssues.md](https://github.com/LuxoftAKutsan/sdl_atf_test_scripts/blob/master/KnownIssues.md)
+#### Known Issues
+Some test cases are failed due to known SDL issues. List of failed test cases avaliable in
+[KnownIssues.md](https://github.com/LuxoftAKutsan/sdl_atf_test_scripts/blob/master/KnownIssues.md)
```

Comment:
@LuxoftAKutsan, link to known issues still doesn't work. Since you don't like the idea of the wiki page for this purpose, let's just keep the name and remove the link.